### DISCUSSION TITLE
Default actions not showing?

### DISCUSSION THREAD

**Discussion by AlanMcCann** [naymspace/backpex#518] [Q&A] [answered]
I'm not able to see the default set of actions on the list/index view. Inspecting the html shows a div for the actions but no contents.  I figure I am missing something pretty obvious in the configuration. Any guidance is appreciated.

Here is a sample live resource

```
defmodule MyAppWeb.Live.DatasetLive do
  @moduledoc false
  use Backpex.LiveResource,
    layout: {MyAppWeb.Layouts, :admin},
    schema: MyApp.Datasets.Dataset,
    repo: MyApp.Repo,
    update_changeset: &MyApp.Datasets.Dataset.update_changeset/3,
    create_changeset: &MyApp.Datasets.Dataset.create_changeset/3,
    pubsub: MyApp.PubSub,
    topic: "datasets",
    event_prefix: "dataset_"

  alias Explorer.Datasets

  @impl Backpex.LiveResource
  def singular_name, do: "Dataset"

  @impl Backpex.LiveResource
  def plural_name, do: "Datasets"

  @impl Backpex.LiveResource
  def fields do
    [
      name: %{
        module: Backpex.Fields.Text,
        label: "Name"
      },
      type: %{
        module: Backpex.Fields.Select,
        label: "Type",
        options: [Source: "source", Synthetic: "synthetic"]
      },
      synthetic_datasets: %{
        module: Backpex.Fields.HasMany,
        label: "Synthetic Datasets",
        display_field: :name,
        # options_query: &where(&1, [user], user.role == :admin),
        live_resource: MyAppWeb.Live.SyntheticDatasetLive
      },
      org: %{
        module: Backpex.Fields.BelongsTo,
        label: "Organization",
        display_field: :name,
        live_resource: MyAppWeb.Live.OrgLive
      }
    ]
  end
end
```

**Comment by Flo0807** [comment]
Hey @AlanMcCann, apologies for the delayed response!

According to your Live Resource configuration, I cannot see any reasons why your resource actions are not showing up.

I went to `https://backpex.live/` and inspected the HTML of the Edit Item Action. It should look something like this:

```html
<div class="tooltip hover:z-30" data-tip="Edit">
  <button type="button" phx-click="item-action" phx-value-action-key="edit" phx-value-item-id="1" aria-label="Edit">
    <span class="hero-pencil-square h-5 w-5 cursor-pointer transition duration-75 hover:scale-110 hover:text-blue-600" aria-hidden="true" fill="currentColor" viewbox="0 0 24 24"></span>
  </button>
</div>
```

This `div` is empty on your page?

Do you have installed Heroicons correctly on your page? If you insert `<Backpex.HTML.CoreComponents.icon name="hero-user" class="h-5 w-5 bg-red-800"/>` on your page, is the icon showing up?

**Comment by AlanMcCann** [comment, marked as answer]
Thanks and no worries about response times! That helped me get on the right path.

This was my fault - using another library requiring a conflicting version of heroicons.

I was able to create a workaround for now by explicitly calling out heroicons as heroicons2 in mix.exs

```
  [
    ...
    {:heroicons2,
      github: "tailwindlabs/heroicons",
      tag: "v2.1.1",
      sparse: "optimized",
      app: false,
      compile: false,
      depth: 1}
  ]
```
  
  and then I changed one line in the tailwind config and everything worked.
  
  ```
        let iconsDir = path.join(__dirname, "../deps/heroicons2/optimized");
```

That got me going for now with while the other library catches up.
